libs/ui/src/unconfigure_machine_button.tsx

```diff
@@ -70,7 +70,7 @@ export function UnconfigureMachineButton({
               <Loading>Unconfiguring machine</Loading>
             ) : (
               <React.Fragment>
-                <H2 as="h1">Delete all election data?</H2>
+                <H2 as="h1">Unconfigure Machine</H2>
```

Comment:
@mattroe I don't know if we were very specific on this one so tagging you.

I like the pattern of the modal title matching the button, but we have a reason (called out in code comments) to have the button text be different here which makes sense. Just removed the "Yes, " pattern.

<img width="554" alt="Screen Shot 2024-10-09 at 5 19 59 PM" src="https://github.com/user-attachments/assets/68dca68c-8ab8-4bf6-81e5-35dfebe670a9">
<img width="547" alt="Screen Shot 2024-10-09 at 5 22 04 PM" src="https://github.com/user-attachments/assets/0c989753-6d27-4de8-b3cb-bc6d0dc865e5">

Reply:
ok wfm